addons/survey/static/src/js/survey_form.js

```diff
@@ -7,6 +7,7 @@ import { scrollTo } from "@web_editor/js/common/scrolling";
 
```

Comment:
Commit message:

You can remove the reference to "test_website_slides_full", it's not the important part, that way we keep the header in a single line.